src/apollo/global/globalTypes.json

```diff
@@ -1,17 +1,25 @@
 {
 	"operations": [
 		{
-			"filePath": "file:///Users/yelpadillo/workspace/work/myetherwalletv6/src/apollo/queries/notifications/notification.graphql",
+			"filePath": "file:///Users/jessicapeng/MyEtherWallet/src/apollo/queries/notifications/notification.graphql",
```

Comment:
this has user path info, I dont think we should have this file

Reply:
@olgakup idk why it does it like this D: i think its cause its js instead of ts...since we manually write global types is this still needed ?